src/main/java/app/model/Grid/AcoGrid.java

```diff
@@ -40,5 +40,17 @@ public void evaporationProcess()
         }
     }
 
+    public void displayGridState()
```

Comment:
might be better if it was just called `print()` ?

Alternatively consider returning the grid state as a string and overriding the `toString()` method.  This has the added bonus of being rendered when you debug, rather than just seeing an object memory reference.